### DISCUSSION TITLE
What are the actual requirements to get the YOLO achievement?

### DISCUSSION THREAD

**Discussion by FleetAdmiralJakob** [community/community#39281] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Because I don't get it and I want to know what I have exactly to do 😅

**Comment by radar-blip** [comment]
See selected answer here: #19146

More info on all achievements here: [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)

**Reply by FleetAdmiralJakob** [reply]
But there are several PRs that want to change the Yolo Achievement Description, because it's not working and I want to know the exactly requirements.

**Reply by radar-blip** [reply]
Just open a pull request in one of your own repositories and merge it (without reviewing it).

**Reply by OtabekgithubJava** [reply]
yes

**Reply by AlvarezMar** [reply]
But I always need someone to review my commits before it can be merged

**Reply by shihabuddin-dev** [reply]
great

**Comment by DSDmark** [comment]
Hi @FleetAdmiralJakob,

I think there is such an explicit mention of how to get that **Achievement**.

> Merged a pull request without code review

So, just open any **repository** and edit what you want and after that create a branch and commit your branch now create a pull request and merged it like that.

- ![2022-11-17_15-26](https://user-images.githubusercontent.com/85744816/202383093-9f442447-a564-47cf-9581-0b53ad3dc03b.png)
- ![2022-11-17_15-27](https://user-images.githubusercontent.com/85744816/202383115-3f8b9a81-c521-4f75-8b9f-bfb7d64b7da7.png)
- ![2022-11-17_15-28](https://user-images.githubusercontent.com/85744816/202383123-d38cdd85-ec70-427b-a708-c04cee1c3c50.png)

**Comment by sahannimesh99** [comment]
I assume this achievement is not achievable now. Because I have merged many times without reviewing my own PRs. But there is no sign of YOLO achievement.

**Reply by radar-blip** [reply]
The YOLO achievement is under your profile picture.

**Comment by nsafarova** [comment, marked as answer]
1. Create a new pull request in your repo and open it
2. In the top right corner at Reviewers section, request someone to review your pull request
3. Merge the pull request without finishing the review.

**Reply by 0xTrip** [reply]
♥

**Reply by codetechie-G** [reply]
Awesome,
Yolo achievement is created immediately.

**Reply by Ramedon1** [reply]
♥

**Reply by rodrigcasio** [reply]
thank you !
<3

**Reply by shohagcsediu** [reply]
thank you, it works

**Reply by ghubliming** [reply]
Thanks, I yoloed

**Comment by wish9** [comment]
The YOLO achievement happened for me when a PR was merged for the first time! Just for your reference!

**Reply by Ramedon1** [reply]
♥

**Comment by YasinDehfuli** [comment]
This course will show you step by step how to get github Yolo badge : 

https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.yolo.md

**Comment by prusrafal** [comment]
Sure, here are simpler steps to follow:

1. Make changes to your project.
2. Send those changes to your GitHub account.
3. Ask someone to check your changes by opening a review request.
4. While waiting for their feedback, include your changes in the main project.
5. Once your changes are part of the main project, let them know they can review your work.
6. Once they approve it, your changes are officially added to the main project.

**Comment by RidmaPalansuriya** [comment]
### **Hi Friends, here's a step-by-step guide to achieving the YOLO achievement**


**1.	Create a GitHub Repository**
Start by creating a repository and make sure it's set to allow multiple contributors.


> [!TIP]
> Helpful advice for doing things better or more easily. 

**2.	Add Contributors (at least one)**
Invite at least one other contributor to your repository.  (Invite an account that you have access to the repository will work too)
You can do this by going to the repository settings and finding the "Collaborators" section.


![00](https://github.com/community/community/assets/109016403/aa1059d4-0a3d-40b4-b594-f0891c786f97)




**3.	Make Changes and Commit Changes**
Make some changes to the repository. (It could be adding a new file, modifying an existing one, or anything else you would like.)


![2](https://github.com/community/community/assets/109016403/3281e95f-c307-4d44-8d6b-15096c99d765)



Once you have made your changes, commit them to your branch. Write a descriptive commit message explaining what you have done is a good practice.



**4.	Create a Pull Request**
After committing your changes, go to the repository's page on GitHub and click the "Pull Request" button. 
Compare your branch to the main branch and create the pull request.

![3](https://github.com/community/community/assets/109016403/e8a2cacb-a5e3-4c4c-9e6b-b7d2be079a31)


Give your pull request a meaningful title that summarizes the changes you've made. If necessary, further explain the purpose of your changes.



**5.	Merge the Pull Request**
Once the pull request is created, review it to ensure everything looks good. Then, click the "Merge Pull Request" button to merge your changes into the main branch.

![5](https://github.com/community/community/assets/109016403/f540c22c-fa6d-4646-a9ac-a0c0eae44354)




Confirm the merge by selecting your email address and clicking the “Confirm merge” button. This finalizes the process.

![6](https://github.com/community/community/assets/109016403/67b47103-ba98-49ad-b909-82d24672df0f)





### That's It! 
_Congratulations, you've successfully achieved the YOLO achievement by collaborating with others and making changes to a repository._

**Comment by msrwn** [comment]
FYI, in my case, the repository should be public, not private. I hope it's helpful for you.

**Comment by longyijdos** [comment]
What worked for me:
1. Opened my own PR
2. Requested a collaborator as reviewer
3. Merged it before they submitted any review
